cmd/validator/flags/flags.go

```diff
@@ -322,6 +322,26 @@ var (
 		Usage: "Enables more verbose logging for counting down to duty",
 		Value: false,
 	}
+
+	// ValidatorsProposerConfigDirFlag defines the path or URL to a file with proposer config.
+	ValidatorsProposerConfigDirFlag = &cli.StringFlag{
+		Name:  "validators-proposer-config-dir",
+		Usage: "remote local file path to load proposer configuration from",
```

Comment:
This should have more info on the usage (see below). The flag is also called config dir but usage says it should be a file path. We should also mention what kind of file (json or yaml?)